{"content": {"suites": {"0": {"status": {"total_pass": 0, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 2, "total_error": 0}, "tests": {"0": {"status": "FAIL", "message": "   TypeError: TestDatabase.test_database_connection() takes 0 positional arguments but 1 was given\n", "test_name": "test_database_connection", "rerun": "0"}, "1": {"status": "FAIL", "message": "   TypeError: TestDatabase.test_data_insertion() takes 0 positional arguments but 1 was given\n", "test_name": "test_data_insertion", "rerun": "0"}}, "suite_name": "tests/test_db.py"}, "1": {"suite_name": "tests/test_garage_page.py", "tests": {"0": {"status": "PASS", "message": "", "test_name": "test_add_car", "rerun": "0"}}, "status": {"total_pass": 1, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 0, "total_error": 0}}, "2": {"suite_name": "tests/test_hw_29_postgres.py", "tests": {"0": {"status": "PASS", "message": "", "test_name": "test_database_connection", "rerun": "0"}, "1": {"status": "PASS", "message": "", "test_name": "test_data_insertion", "rerun": "0"}, "2": {"status": "PASS", "message": "", "test_name": "test_fetch_users", "rerun": "0"}, "3": {"status": "PASS", "message": "", "test_name": "test_update_data", "rerun": "0"}, "4": {"status": "PASS", "message": "", "test_name": "test_delete_data", "rerun": "0"}}, "status": {"total_pass": 5, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 0, "total_error": 0}}, "3": {"suite_name": "tests/test_hw_29_postgres1.py", "tests": {"0": {"status": "PASS", "message": "", "test_name": "test_database_connection", "rerun": "0"}, "1": {"status": "PASS", "message": "", "test_name": "test_data_insertion[1-John]", "rerun": "0"}, "2": {"status": "FAIL", "message": "               psycopg2.errors.UniqueViolation: duplicate key value violates unique constraint \"users_pkey\"\n               DETAIL:  Key (id)=(2) already exists.\n", "test_name": "test_data_insertion[2-Jane]", "rerun": "0"}, "3": {"status": "FAIL", "message": "               psycopg2.errors.UniqueViolation: duplicate key value violates unique constraint \"users_pkey\"\n               DETAIL:  Key (id)=(3) already exists.\n", "test_name": "test_data_insertion[3-Alice]", "rerun": "0"}, "4": {"status": "PASS", "message": "", "test_name": "test_fetch_users", "rerun": "0"}}, "status": {"total_pass": 3, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 2, "total_error": 0}}, "4": {"suite_name": "tests/test_instructions_page.py", "tests": {"0": {"status": "SKIP", "message": "('C:\\\\Users\\\\feshv\\\\hillel_py\\\\aqa_hillel_13_08\\\\lesson_29_separate_project\\\\tests\\\\test_instructions_page.py', 14, 'Skipped: Not completed')\n", "test_name": "test_instructions_download", "rerun": "0"}}, "status": {"total_pass": 0, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 0, "total_error": 0}}, "5": {"suite_name": "tests/test_postgres_in_docker.py", "tests": {"0": {"status": "PASS", "message": "", "test_name": "test_database_connection", "rerun": "0"}, "1": {"status": "FAIL", "message": "   psycopg2.errors.UniqueViolation: duplicate key value violates unique constraint \"users_pkey\"\n   DETAIL:  Key (id)=(1) already exists.\n", "test_name": "test_data_insertion", "rerun": "0"}}, "status": {"total_pass": 1, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 1, "total_error": 0}}}}, "date": "December 09, 2024", "start_time": 1733770304.9288955, "total_suite": 6, "status": "FAIL", "status_list": {"pass": "10", "fail": "5", "skip": "0", "error": "0", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "15"}